contributors/guide/README.md

```diff
@@ -173,10 +173,19 @@ The pull request workflow is described in detail [here](/contributors/devel/pull
 
 ## Code Review
 
-_Improvements needed_
-* Clarify and streamline some of the explanations from [here](https://git.k8s.io/community/contributors/devel/owners.md#code-review-process).
+For a brief description of the importance of code review, please read [On Code Review](/contributors/devel/community-expectations.md#code-review).  There are two aspects of code review: giving and receiving.
+
+To make it easier for your PR to receive reviews, consider the reviewers will need you to:
+
+* write [good commit messages](https://chris.beams.io/posts/git-commit/)
+* break large changes into a logical series of smaller patches which individually make easily understandable changes, and in aggregate solve a broader issue
+* label PRs with appropriate SIGs and reviewers: to do this first look in the source tree for `OWNERS` files at and above the level where you are making changes.  The [OWNERS files documentation](/contributors/devel/owners.md) describes in more detail how to find and read these files.  Then enter into issue comments bot commands like `/sig docs` and `/assign zacharysarah` to for example to label for sig-docs and assign to Zach Corleissen by github id.
```

Comment:
As far as I understand it, assigning reviewers and approvers is automated and I don't think we should encourage contributors to assign reviewers themselves.

Reply:
The bot suggests one do so.  Eg: above in this PR is:

https://github.com/kubernetes/community/pull/1496#issuecomment-351222798

Reply:
I saw that. But this place says the doc will find them automatically. https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md#code-review-process - so does this: https://github.com/kubernetes/community/blob/master/contributors/devel/pull-requests.md#the-testing-and-merge-workflow

Reply:
we should assume that it is automatic. perhaps the bot text should be altered to "if you do not see a reviewer, please assign <suggested person>.

Reply:
I think that is the right approach @cblecker. I'm a bit fuzzy on what the bot does when you did not add a SIG however. Does it suggest a specific one based on the repo you're in?

Reply:
suggestion: skip any mention of OWNERS files in this process (that is for reviewers) and instead reference the bot commands a bit more clearly in /contributors/devel/pull-requests, as well say here to follow bot instructions.

Reply:
It doesn't right now, but adding a sig based on the code isn't something we require/enforce today.

Reply:
Changing that third bullet to:

* label PRs with appropriate SIGs and reviewers: to do this read the messages the bot sends you to guide you through the PR process

Reply:
I have to say the bot's choice of auto-assignees for this issue was surprising and I suspected wrong, but...I just went with it.  When it does NOT auto-assign, I've been trying to assign, but that is actually not a newb expected action versus something for people who do triage.  I like the simplified wording here.  Go with the bot and trust in the system.